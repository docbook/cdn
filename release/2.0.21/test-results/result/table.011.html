<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for table.011.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: table.011.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">9</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;p&gt;Simple cell spacing test. The following single-column table
should show both cell contents starting at the same place.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;CALS:&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;figure <span class="att">class="informaltable-wrapper"</span> <span class="att">id="R.1.4"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="informaltable"</span>&gt;
               <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "</span>&gt;
                  <span class="tag">&lt;colgroup&gt;
                     <span class="tag">&lt;col <span class="att">class="tcol1"</span> <span class="att">width="100%"</span>/&gt;</span>
                  &lt;/colgroup&gt;</span>
                  <span class="tag">&lt;tbody&gt;
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td <span class="att">style="border-bottom: 1px solid ; "</span>&gt;First entry.&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td&gt;
Second entry.&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                  &lt;/tbody&gt;</span>
               &lt;/table&gt;</span>
            &lt;/div&gt;</span>
         &lt;/figure&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;HTML:&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;figure <span class="att">class="informaltable-wrapper"</span> <span class="att">id="R.1.6"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="informaltable"</span>&gt;
               <span class="tag">&lt;table <span class="att">border="1"</span>&gt;
                  <span class="tag">&lt;tbody&gt;
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td&gt;First entry.&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td&gt;
Second entry.&lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                  &lt;/tbody&gt;</span>
               &lt;/table&gt;</span>
            &lt;/div&gt;</span>
         &lt;/figure&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;p&gt;Simple cell spacing test. The following single-column table
should show both cell contents starting at the same place.&lt;/p&gt;</span>
            <span class="tag">&lt;p&gt;CALS:&lt;/p&gt;</span>
            <span class="tag">&lt;figure <span class="att">class="informaltable-wrapper"</span> <span class="att">id="R.1.4"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="informaltable"</span>&gt;
                  <span class="tag">&lt;table <span class="att">border="0"</span> <span class="att">style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "</span>&gt;
                     <span class="tag">&lt;colgroup&gt;
                        <span class="tag">&lt;col <span class="att">class="tcol1"</span> <span class="att">width="100%"</span>/&gt;</span>
                     &lt;/colgroup&gt;</span>
                     <span class="tag">&lt;tbody&gt;
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td <span class="att">style="border-bottom: 1px solid ; "</span>&gt;First entry.&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td&gt;
Second entry.&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                     &lt;/tbody&gt;</span>
                  &lt;/table&gt;</span>
               &lt;/div&gt;</span>
            &lt;/figure&gt;</span>
            <span class="tag">&lt;p&gt;HTML:&lt;/p&gt;</span>
            <span class="tag">&lt;figure <span class="att">class="informaltable-wrapper"</span> <span class="att">id="R.1.6"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="informaltable"</span>&gt;
                  <span class="tag">&lt;table <span class="att">border="1"</span>&gt;
                     <span class="tag">&lt;tbody&gt;
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td&gt;First entry.&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td&gt;
Second entry.&lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                     &lt;/tbody&gt;</span>
                  &lt;/table&gt;</span>
               &lt;/div&gt;</span>
            &lt;/figure&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>Simple cell spacing test. The following single-column table
should show both cell contents starting at the same place.</p><p>CALS:</p><figure id="R.1.4" class="informaltable-wrapper"><div class="informaltable"><table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="100%" /></colgroup><tbody><tr><td style="border-bottom: 1px solid ; ">First entry.</td></tr><tr><td>
Second entry.</td></tr></tbody></table></div></figure><p>HTML:</p><figure id="R.1.6" class="informaltable-wrapper"><div class="informaltable"><table border="1"><tbody><tr>
          <td>First entry.</td>
        </tr><tr>
          <td>
Second entry.</td>
        </tr></tbody></table></div></figure></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="content"><p>Simple cell spacing test. The following single-column table
should show both cell contents starting at the same place.</p><p>CALS:</p><figure id="R.1.4" class="informaltable-wrapper"><div class="informaltable"><table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="100%" /></colgroup><tbody><tr><td style="border-bottom: 1px solid ; ">First entry.</td></tr><tr><td>
Second entry.</td></tr></tbody></table></div></figure><p>HTML:</p><figure id="R.1.6" class="informaltable-wrapper"><div class="informaltable"><table border="1"><tbody><tr>
          <td>First entry.</td>
        </tr><tr>
          <td>
Second entry.</td>
        </tr></tbody></table></div></figure></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/table.011.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>