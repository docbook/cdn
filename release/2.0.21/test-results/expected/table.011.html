<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>Simple cell spacing test. The following single-column table
should show both cell contents starting at the same place.</p><p>CALS:</p><figure id="R.1.4" class="informaltable-wrapper"><div class="informaltable"><table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="100%" /></colgroup><tbody><tr><td style="border-bottom: 1px solid ; ">First entry.</td></tr><tr><td>
Second entry.</td></tr></tbody></table></div></figure><p>HTML:</p><figure id="R.1.6" class="informaltable-wrapper"><div class="informaltable"><table border="1"><tbody><tr>
          <td>First entry.</td>
        </tr><tr>
          <td>
Second entry.</td>
        </tr></tbody></table></div></figure></article><script src="../../resources/js/prism.js"></script></body></html>